images/kube-webhook-certgen/README.md

```diff
@@ -0,0 +1,13 @@
+# Grafana v12.1.0
```

Comment:
why is this called grafana? this is suppose to be kube-webhook-certgen right? and also write what it this image about and where is this consumed?